src/test/resources/features/decs/ManagementUI.feature

```diff
@@ -359,4 +359,20 @@ Feature: ManagementUI
     And I create a "MIN" case with "Cyber Stalking And Harassment" as the primary topic
     And I override the initial draft team of the case to the team created in Management UI
     And I load the current case
-    Then the case should be assigned to the DCU draft team created in Management UI
\ No newline at end of file
+    Then the case should be assigned to the DCU draft team created in Management UI
+
+  @TeamManagement @Regression
+  Scenario: User is able to move a team to a different unit
+    Given I navigate to the "Team" Management page
+    When I select the "Detention Services" team from the dropdown
+    And I navigate to edit team
+    And I change to a different unit
```

Comment:
I think it would be better to split the last part into 2 steps, one for actioning the change of the unit, and then an assertion step for checking for the success message.